.github/workflows/release.yml

```diff
@@ -108,8 +109,7 @@ jobs:
       if: ${{ matrix.test == 'helm' }}
       run: |
           projectdir=$(pwd)
-          shorttag="${TAG#v}"
-          go test --failfast --mod=readonly "./test" --linkerd="$HOME/.linkerd2/bin/linkerd" --helm-path="$projectdir/bin/helm" --smi-helm-chart="$projectdir/build-archives/helm/linkerd-smi-$shorttag.tgz" --integration-tests
+          go test --failfast --mod=readonly "./test" --linkerd="$HOME/.linkerd2/bin/linkerd" --helm-path="$projectdir/bin/helm" --smi-helm-chart="$projectdir/build-archives/helm/linkerd-smi-$CHART_VERSION.tgz" --integration-tests
```

Comment:
nit, take it or leave it: i might want to put line breaks in this. i know it wasn't previously like that though.